airflow/dags/ingestion.py

```diff
@@ -13,19 +13,45 @@
 data_ingestion_dag = DAG(
     'data_ingestion_dag',
     default_args=default_args,
+    schedule_interval="@weekly",  # Run once a week at midnight on Sunday morning
```

Comment:
Double checked the update frequency (https://www.cdc.gov/nchs/covid19/covid-19-mortality-data-files.htm) and given that, I'd change this to run once at around 1PM Friday, and once around 6PM Wednesday to get as close as we can to their updates, unless I'm looking at the wrong data source (decent chance I could be, in which case, ignore this).

Reply:
Doh, I was looking at the wrong one. How often does this dataset update, and on which days of the week/time?

Reply:
I did the ingestion for the prototype. If we run it daily we will need logic to avoid appending duplicate data. Doable but still needs to be implemented.

Reply:
Sounds good. I'll leave this part commented out until we address #30.